src/backoffice/settings/dashboards/health-check/views/health-check-group.element.ts

```diff
@@ -0,0 +1,204 @@
+			}
+
+			p {
+				margin: 0;
+			}
+
+			.header {
+				display: flex;
+				justify-content: space-between;
+				align-items: center;
+			}
+
+			.check-results-wrapper {
+				margin-top: var(--uui-size-space-5);
+			}
+
+			.check-results-wrapper .check-result:not(:first-child) {
+				padding-top: var(--uui-size-space-5);
+				border-top: 1px solid var(--uui-color-divider-standalone);
+			}
+
+			.check-results-wrapper uui-button {
+				margin-block-start: 1em;
+			}
+
+			.check-result-description {
+				display: flex;
+			}
+
+			.check-result-description span {
+				width: 36px;
+			}
+
+			uui-icon {
+				vertical-align: sub;
+			}
+		`,
+	];
+
+	@property()
+	groupName!: string;
+
+	@state()
+	private _buttonState: UUIButtonState;
+
+	@state()
+	private _group?: HealthCheckGroupWithResult;
+
+	private _healthCheckContext?: UmbHealthCheckDashboardContext;
+
+	@state()
+	private _checks?: HealthCheckWithResult[] | null;
+
+	@state()
+	private _keyResults?: any;
+
+	private _api?: UmbHealthCheckContext;
+
+	constructor() {
+		super();
+		this.consumeContext<UmbHealthCheckDashboardContext>('umbHealthCheckDashboard', (instance) => {
+			this._healthCheckContext = instance;
+
+			this._api = this._healthCheckContext?.apis.get(this.groupName);
+
+			this._api?.getGroupChecks(this.groupName);
+
+			this._api?.checks.subscribe((checks) => {
+				this._checks = checks;
+				this._group = { name: this.groupName, checks: this._checks };
+			});
+
+			this._api?.results.subscribe((results) => {
+				this._keyResults = results;
+			});
+		});
+	}
+
+	private async _buttonHandler() {
+		this._buttonState = 'waiting';
+		this._api?.checkGroup(this.groupName);
+		this._buttonState = 'success';
+	}
+
+	private _onActionClick(action: HealthCheckAction) {
+		return tryExecuteAndNotify(this, HealthCheckResource.postHealthCheckExecuteAction({ requestBody: action }));
+	}
+
+	render() {
+		if (this._group) {
+			return html`
+				<div class="header">
+					<h2>${this._group.name}</h2>
+					<uui-button
+						label="Get checks"
+						color="positive"
+						look="primary"
+						.state="${this._buttonState}"
+						@click="${this._buttonHandler}">
+						Get checks
+					</uui-button>
+				</div>
+				<div class="checks-wrapper">
+					${this._group.checks?.map((check) => {
+						return html`<uui-box headline="${check.name || '?'}">
+							<p>${check.description}</p>
+							${check.key ? this.renderCheckResults(check.key) : nothing}
+						</uui-box>`;
+					})}
+				</div>
+			`;
+		} else return nothing;
+	}
+
+	renderCheckResults(key: string) {
+		const checkResults = this._keyResults?.find((result: any) => result.key === key);
+		return html`<uui-icon-registry-essential>
+			<div class="check-results-wrapper">
+				${checkResults?.results.map((result: any) => {
+					return html`<div class="check-result">
+						<div class="check-result-description">
+							<span>${this.renderIcon(result.resultType)}</span>
+							<p>${result.message}</p>
```

Comment:
Running this on the actual backend reveals, that the messages may contain raw HTML, which we do not render by default. The question is if we should render it (trust the server) or remove it from the server?

![image](https://user-images.githubusercontent.com/752371/214584537-b0d401cc-29ae-45f0-9b54-269d40196438.png)

Reply:
I think its depends on if the server contains anything important HTML wise. Personally I think its best if we can avoid it from the server.

Reply:
I just had a look at the other messages from the server. They contain some tokens that might be valuable to render appropriately. In truth, we have a lot of HTML in the translation files which we can't get rid of. I fear we have no choice but to trust the server and render it.